src/state/models/ui/preferences.ts

```diff
@@ -13,6 +16,11 @@ import {LANGUAGES} from '../../../locale/languages'
 
 // TEMP we need to permanently convert 'show' to 'ignore', for now we manually convert -prf
 export type LabelPreference = APILabelPreference | 'show'
+export type FeedViewPreference = Omit<AppBskyActorDefs.FeedViewPref, '$type'>
+export type ThreadViewPreference = Omit<
+  AppBskyActorDefs.ThreadViewPref,
+  '$type'
+>
```

Comment:
These types are resulting in `unknown`s, unsure why it's flattening to it's most basic type value 🤔

Reply:
Yeah it's bugging me too. You know what, gonna manually spec it locally.

Reply:
Yeah it seems to be the `Omit<>` modifier that ruins it

Reply:
https://github.com/bluesky-social/atproto/pull/1653